config/routes.rb

```diff
@@ -20,6 +20,12 @@ def scoped_navigation_routes(context, navigation, as_redirects: false)
   end
 
   mount Cfa::Styleguide::Engine => "/cfa"
+
+  # In order to disambiguate versions of english pages with and without locales, we redirect to URLs including the locale
+  # If we redirect here in the route declarations, we can't inspect accept headers to determine the proper default locale,
+  # hence the redirect actions in public_pages.
+  root "public_pages#redirect_locale_home", as: :redirected_root
```

Comment:
putting this before the locale scope allows us to keep the locale as optional within the locale scope, which is a much less intrusive change than having to redirect all urls without locales to the version with locales.